package.json

```diff
@@ -80,6 +80,7 @@
     "@typescript-eslint/eslint-plugin": "^2.24.0",
     "@typescript-eslint/parser": "^2.24.0",
     "@zeit-ui/prettier-config": "^1.0.0",
+    "@zeit-ui/react": "^1.8.0-canary.8",
```

Comment:
Why do we depend on this

Reply:
walk around build error, it is possibly not necessary now after change the library dependency to ours